demisto_sdk/commands/format/update_integration.py

```diff
@@ -133,41 +133,28 @@ def set_reputation_commands_basic_argument_as_needed(self):
         )
 
         integration_commands = self.data.get("script", {}).get("commands", [])
-
+        
         for command in integration_commands:
             command_name = command.get("name", "")
 
             if command_name in BANG_COMMAND_NAMES:
-                for argument in command.get(
-                    "arguments", []
-                ):  # If there're arguments under the command
-                    name = argument.get("name")
-                    if name == command_name:
-                        is_array = argument.get("isArray", False)
-                        if not is_array:
-                            logger.info(
-                                f"isArray field in {name} command is set to False. Fix the command to support that function and set it to True."
-                            )
-                        argument.update(
-                            {"default": True, "isArray": is_array, "required": True}
-                        )
-                        break
-                else:  # No arguments at all
-                    default_bang_args = {
-                        "default": True,
-                        "description": "",
-                        "isArray": True,
-                        "name": command_name,
-                        "required": True,
-                        "secret": False,
-                    }
-                    logger.info(
-                        f"Command {command_name} has no arguemnts. Setting them: {json.dumps(default_bang_args, indent=4)}"
-                    )
-                    argument_list: list = command.get("arguments", [])
-                    argument_list.append(default_bang_args)
-                    command["arguments"] = argument_list
-
+                
+                arguments =  command.get("arguments")
+                if not arguments:  # if no argument in the command
+                    return
+                
+                if any(argument.get('default') for argument in arguments):  # if one of the arguments already have a default value
+                    return
+                
+                default_argument =  {"default": True, "required": True}
+  
+                # if one of the arguments have the same name as command name, update him to be a default
+                for argument in arguments:
+                    if argument.get("name", "") == command_name:
```

Comment:
```suggestion
                    if argument["name"] == command_name:
```
it's a guaranteed field, no need for a `get`